data/mods/CRT_EXPANSION/items/crt_ammo.json

```
@@ -3,7 +3,7 @@
     "type": "AMMO",
     "id": "pellet",
     "price": 1000,
-    "name": "lead pellets",
+    "name": { "str": "lead pellets" },
```

```suggestion
    "name": { "str_sp": "lead pellets" },
```
And the same for other pellets.